content/blog/2024-11-19-november-GREI/contents.lr

```diff
@@ -0,0 +1,58 @@
+
+The plans for the third year of GREI have been nearly fully implemented, and the group that gathered in Chicago last month reported on outcomes and completed planning exercises for priorities for Year 4. The Year 3 plan is [published and available on GitHub](https://medium.com/@blog-grei/unlocking-innovation-the-generalist-repositories-embrace-transparency-and-community-on-github-08cdef676e2e), and is broken down into eight objectives. Below is an outline of how Zenodo representatives contributed to each objective, foundational work which often aligned with improvements being made for the InvenioRDM development and user communities:
+
+### Desirable Characteristics for All Data Repositories (for sharing scientific data)
+
+- Zenodo team members co-led the effort to update the [Generalist Repository Comparison Chart](https://doi.org/10.5281/zenodo.7946938), a tool to help researchers select the best generalist repository to meet their needs. The new version is projected for Spring 2025
+- Zenodo team members co-led the team which created the [Generalist Repository Selection Flowchart](https://doi.org/10.5281/zenodo.11105430), a tool designed to guide users through a series of considerations while selecting the best repository to share data
+
+### Consistent Metadata
+
+- Zenodo and InvenioRDM continue to adhere to the recommended GREI standard, which incorporates the DataCite metadata schema. In addition, InvenioRDM already has capability for further suggested metadata enhancements, such as incorporation of LCSH and MeSH subject headings and CRediT [contributor role taxonomy](https://credit.niso.org/) roles
+
+### Search & Browse
+
+- Work is ongoing to explore a cross-GREI-repository search option. Meanwhile, InvenioRDM has already achieved a further recommendation: incorporation of [ROR](https://ror.org/), the identification registry for research organizations
+
+### Analytics & Reporting
+
+- Zenodo team members are co-leading the effort to share the [Make Data Count](https://makedatacount.org/learn-about-us/) usage metrics with the wider community via DataCite 
+
+### Use Cases
+
+- Zenodo team members co-led the task group responsible for updating the [Use Case catalog](https://doi.org/10.5281/zenodo.12212654) with two new use cases: one outlining a scenario where a researcher utilizes a generalist repository in lieu of a local institutional repository, and one scenario where a researcher must deposit portions of a heterogenous dataset into several different repositories, one of which is a generalist repository
+
+### Connecting Digital Objects
+
+- Zenodo team members are working on the effort to map DataCite relationTypes to all types of non-data research objects, and to submit any needed updates to DataCite via API
+- InvenioRDM has also completed work on the FAIR signposting recommendation, thanks to Guillaume Viger’s work on [signposting](https://doi.org/10.5281/zenodo.12554416)
```

Comment:
Does it make sense to add a link to `Guillaume Viger’s` words to some public bio page (or ORCID?), if such bio page exist?